plugins/fe_events_moodle/lang/en/local_fe_events_direct.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
This file is named incorrectly and is why the plugin name doesn't appear properly in the admin view:

![image](https://user-images.githubusercontent.com/839025/180849449-877c2414-da68-4f4f-9a22-d397326db33e.png)

Try changing to `local_fe_events_moodle.php`.